### PR TITLE
Word embedding and time zone counts applications

### PR DESCRIPTION
**Pull request by ljj7975**

### CODE REVIEW

File: src/main/scala/ca/uwaterloo/SIGIR/TimeZoneCount.scala [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        try {
+          val pair:Tuple2[String, Int] = ((parsedJson \ "user" \ "time_zone").as[String], 1)
+          timeZone = List(pair)
+        } catch {
+          case e : Exception => {
+            System.out.println("field time_zone unavailble for the following tweet")
+            println(Json.prettyPrint(parsedJson))
+          }
+        }
+        timeZone
```
Comment (zeynepakkalyoncu) [inline src/main/scala/ca/uwaterloo/SIGIR/TimeZoneCount.scala]: What exactly is at index 1 and 2 of timeZone again?

Comment (ljj7975) [reply src/main/scala/ca/uwaterloo/SIGIR/TimeZoneCount.scala]: index1 is timezone index2 is count as in line 37

File: src/main/scala/ca/uwaterloo/SIGIR/WordEmbedding.scala [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      .query(field + ":" + term)
+      .map(doc => {
+        val parsedJson = Json.parse(doc.get(field).toString)
+        val contents = Stemmer.stem(parsedJson("text").toString)
+        contents.toString.split(" ").toSeq
+      })
+
+    val model = word2vec.fit(rdd)
+
+    val synonyms = model.findSynonyms(term, 10)
```
[the remark this answers is not in the document]
Comment (ljj7975) [reply src/main/scala/ca/uwaterloo/SIGIR/WordEmbedding.scala]: Yes it does

File: src/main/scala/ca/uwaterloo/SIGIR/TimeZoneCount.scala [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      }).reduceByKey(_+_).sortBy(_._2, false)
+
+    val topTimeZones = rdd.take(10)
+
+    println(s"top 10 cities with the most tweets")
```
[the remark this answers is not in the document]
Comment (ljj7975) [reply src/main/scala/ca/uwaterloo/SIGIR/TimeZoneCount.scala]: will update it in the following diff

**Pull request merged**